codeclimate-wrapper

```diff
@@ -41,17 +41,31 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+: ${XDG_CONFIG_HOME:=$HOME/.config/}
+CC_CONFIG=$(realpath "${CODECLIMATE_CONFIG:-$XDG_CONFIG_HOME/codeclimate/config.yml}")
```

Comment:
`realpath` is defined as a function in POSIX. Also present as a CLI tool in GNU coreutils. It's main use case is expansion of symlinks but it also gives a full path for a relative paths.

But on a second though we don't need it here. Docker mounts relative paths just fine.